yb-voyager/src/queryparser/helpers.go

```diff
@@ -7,9 +7,9 @@ import (
 const (
 	DOCS_LINK_PREFIX        = "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/"
 	POSTGRESQL_PREFIX       = "postgresql/"
-	ADVISORY_LOCKS_DOC_LINK = DOCS_LINK_PREFIX + POSTGRESQL_PREFIX + "#advisory-locks"
-	SYSTEM_COLUMNS_DOC_LINK = DOCS_LINK_PREFIX + POSTGRESQL_PREFIX + "#system-columns"
-	XML_FUNCTIONS_DOC_LINK  = DOCS_LINK_PREFIX + POSTGRESQL_PREFIX + "#xml-functions"
+	ADVISORY_LOCKS_DOC_LINK = DOCS_LINK_PREFIX + POSTGRESQL_PREFIX + "#advisory-locks-is-not-yet-implemented"
```

Comment:
not possible to have it changed on the docs side?

Reply:
I thought it might be better if we put the correct link in the first release other wise if we wanted to change them later it will be this release will have the broken link.
And the heading just `Advisory locks` was not going with structure how we known issue on that page, the title needs to informative about the actual issue